src/draw_functions.py

```diff
@@ -68,7 +68,6 @@ def wave(sample, cr, width, height, colors):
            width / (ls - 1), height_diff * height)
     cr.line_to(width, height)
     cr.line_to(0, height)
-    cr.close_path()
```

Comment:
I guess that Cairo automatically closes path because of `cr.fill()`, so there's no changes. Well, if it works, it works :)

Reply:
I checked several times, it really doesn't affect the Wave mode in any way.